Ginger/Ginger/UserControlsLib/MultiSelectComboBox.xaml.cs

```diff
@@ -333,11 +333,12 @@ private void SetText()
             if (this.SelectedItems != null)
             {
                 StringBuilder displayText = new StringBuilder();
+                bool allSelected = _nodeList.Count(x => x.IsSelected) == _nodeList.Count - 1;
```

Comment:
Please validate this change at other places wherever multibox is used

Reply:
Validated